webapp/channels/src/components/user_settings/display/user_settings_theme/user_settings_theme.test.tsx

```diff
@@ -18,20 +16,6 @@ jest.mock('utils/utils', () => ({
 }));
 
 describe('components/user_settings/display/user_settings_theme/user_settings_theme.jsx', () => {
```

Comment:
```suggestion
describe('components/user_settings/display/user_settings_theme/user_settings_theme', () => {
```

Reply:
Done.

Reply:
I don't think it matters enough to go change everywhere, but I typically just use the component or function name without the path for these because Jest prints the full path in its output